src/model/landmark.js

```diff
@@ -0,0 +1,56 @@
+'use strict';
+
+import mongoose from 'mongoose';
+import HttpError from 'http-errors';
+import Country from './landmark';
```

Comment:
Read this line carefully. Are you importing from the right place? This causes your landmark prehook to fail.